edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/IModule.cs

```diff
@@ -112,8 +112,8 @@ public interface IModule : IEquatable<IModule>
         [JsonProperty(PropertyName = "imagePullPolicy")]
         ImagePullPolicy ImagePullPolicy { get; }
 
-        [JsonProperty(PropertyName = "priority")]
-        uint Priority { get; }
+        [JsonProperty(PropertyName = "startupOrder")]
+        uint StartupOrder { get; }
```

Comment:
I thought we discussed that we only need to change the public facing name of this. The internal name can stay priority (to minimize the changes?)

Reply:
That's the original intent but it ripples through all the various classes and their tests.